### PR TITLE
NO-TICKET Guide/Core doc cleanup

### PR DESCRIPTION
**Pull request by forrestbaer**
## Summary

Updates to documentation at the core repo level.  Added several updates to core LICENSE, README, CONTRIBUTING files for recent loki changes and things that were out of date or changed with the removal of the guides folder.

### Removed

`/guides` - All removed, I don't think any of it is useful presently or was being utilized by others, the docs site should contain these kinds of things in the future

### CODE REVIEW

**Review by pwolfert — COMMENTED** [review]
I agree that most of this should go. The two things that you removed that I'm not sure should be removed just yet, though, are `GOVERNANCE.md` and `GUIDING-PRINCIPLES.md`. I think the governance might be there to fulfill some sort of requirements. Some of the content from `GUIDING-PRINCIPLES` might fit inside `CONTRIBUTING` or something, but I think it might be a little hasty to delete it. I agree with all the other deletions.

**Comment by forrestbaer** [comment]
> I agree that most of this should go. The two things that you removed that I'm not sure should be removed just yet, though, are `GOVERNANCE.md` and `GUIDING-PRINCIPLES.md`. I think the governance might be there to fulfill some sort of requirements. Some of the content from `GUIDING-PRINCIPLES` might fit inside `CONTRIBUTING` or something, but I think it might be a little hasty to delete it. I agree with all the other deletions.

The `GOVERNANCE.md` file just contains a dead link so I'm not sure about that one, I've added the guiding principles to contributing, let me know what you think @pwolfert

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,12 +17,54 @@ Use [our guides](https://github.com/CMSgov/design-system/tree/master/guides) to
...
+
+### Favor generalized names
+
+Avoid naming that indicates a specific use-case or styling, e.g. `snippet` rather than `search-snippet`.
+
+The names and phrasing of the design system's parts should also still make sense when a different theme is applied, e.g. `primary` rather than `blue`.
+
+### Variations exist only when they can't be achieved with a utility class
+
+The design system includes utility classes for modifying the most common visual traits, like spacing, size, and color. You may want to offer variations of a component that involves changing one of these traits. When you do, you should first see if you can accomplish the variation by using one of the utility classes. If you can, then don't create a new CSS ruleset to accomplish something another ruleset already does.
```
Comment (pwolfert) [inline CONTRIBUTING.md:53]:
```suggestion
The design system includes utility classes for modifying the most common visual traits, like spacing, size, and color. You may want to offer variations of a component that involves changing one of these traits. When you do, you should first see if users of the design can accomplish the variation by applying one of the utility classes to your component.
```

File: guides/GOVERNANCE.md [hunk 2]
```diff
@@ -1,2 +0,0 @@
-See our governance process in Confluence
-https://confluence.cms.gov/display/HCDSG/Governance
```
Comment (pwolfert) [inline guides/GOVERNANCE.md:2]: https://confluence.cms.gov/display/HCDSG/Design+system+governance

I use the share button to get the following link, and I'm hoping it means it's a permalink:
https://confluence.cms.gov/x/LYWbGQ

**Review by pwolfert — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (pwolfert) [reply CONTRIBUTING.md:53]: I don't want to encourage people to use utility classes _inside_ of components. That's something we just finished cleaning up!

File: guides/GOVERNANCE.md (hunk 2 above)
Comment (forrestbaer) [reply guides/GOVERNANCE.md:2]: Wondering if it should just go in the README or if it's important that it's in it's own GOVERNANCE file

**Pull request merged**